src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/GlobalErrorHandler.java

```diff
@@ -143,8 +144,7 @@ public Errors handleUpstreamServiceException(final UpstreamServiceException exce
 
   @ResponseStatus(BAD_REQUEST)
   @ExceptionHandler({ValidationException.class, HttpMessageNotReadableException.class,
-      IllegalArgumentException.class, MethodArgumentNotValidException.class,
-      MaxUploadSizeExceededException.class})
+      IllegalArgumentException.class, MethodArgumentNotValidException.class,ValidationException.class})
```

Comment:
`ValidationException.class` was already in the list here

Reply:
👍